backend/app/core/auth/router.py

```diff
@@ -5,8 +5,10 @@
 from app.core.auth.credentials import AWSCredentials
 from app.core.auth.device_flows.router import router as device
 from app.core.auth.service import AuthService
+from app.core.authz.authorization import Authorization
+from app.core.authz.authorization import AuthorizationAction as Action
```

Comment:
You can simplify this import

```suggestion
from app.core.authz import Action, Authorization
```